prep-javascript-variables/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    var fullName = 'Marcio Chiaradia';
+    console.log('value of fullName: ', fullName);
+
+    var numberOfPets = '0';
```

Comment:
Here you can leave out the quotes so that the browser knows that you mean a number and not a string.